libnetwork_internal_test.go

```diff
@@ -8,7 +8,7 @@ import (
 
 func TestDriverRegistration(t *testing.T) {
 	bridgeNetType := "bridge"
-	c := New()
+	c, _ := New()
```

Comment:
Please check the error returned by New() everywhere in the testing code.
A future change could break the current behavior.